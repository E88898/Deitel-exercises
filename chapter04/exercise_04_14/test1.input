100
5394.78
1000
500
5500
-1

