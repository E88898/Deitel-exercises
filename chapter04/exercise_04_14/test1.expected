New balance is 5894
Account: 100
Credit limit: 5500
Balance:      5894.78
Credit limit Exceeded.
